src/FakeItEasy/CallOrderingContext.cs

```diff
@@ -0,0 +1,99 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using FakeItEasy.Core;
+
+    /// <summary>
+    /// Represents a context to make assertions on the order of the calls made on a fake.
+    /// </summary>
+    public class CallOrderingContext : IDisposable

```

Comment:
I don't think so, but I'll give it a try. It probably doesn't matter anyway, since users will never need to call anything on it, just pass it around.